build_modules/lib/src/workers.dart

```diff
@@ -62,7 +62,6 @@ BazelWorkerDriver get _dartdevkDriver {
           p.join(sdkDir, 'bin', 'dart'),
           [
             p.join(sdkDir, 'bin', 'snapshots', 'dartdevc.dart.snapshot'),
-            '--kernel',
```

Comment:
Can/should this go out separately?

Reply:
This is still supported generally, but the entrypoint which compiles the entire SDK does not support it. There isn't really any value in backporting this.